config/rollup.config.js

```diff
@@ -1,8 +1,13 @@
+import fs from 'fs';
 import nodeResolve from 'rollup-plugin-node-resolve';
 import babel from 'rollup-plugin-babel';
 import memory from 'rollup-plugin-memory';
 
+const pkg = JSON.parse(fs.readFileSync('./package.json'));
```

Comment:
I think it works if you load rollup-plugin-json but I mostly copied from preact-compat that uses this line